### ISSUE TITLE
Notification on newly installed app

### ISSUE THREAD

**Issue by jumper047** [pylerSM/BetterBatterySaver#6]
Can you add this? Useful for configuring new apps.

**Comment by pylerSM** [comment]
Yes, so it would open Xposed required per app battery saving settings yes?

**Comment by jumper047** [comment]
Yes. And, also if it is possible, add a button to the app info screen with same function.

25 июля 2016 г. 12:52:56 GMT+03:00, pyler notifications@github.com пишет:

> Yes, so it would open Xposed required per app battery saving settings
> yes?
> 
> ---
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pylerSM/BetterBatterySaver/issues/6#issuecomment-234910503

## 

Простите за краткость, создано в K-9 Mail.

**Comment by pylerSM** [comment]
Button in app info no, I think. But notification is fine.

**Comment by jumper047** [comment]
Good to hear:).About button- https://gitlab.com/sangre/XPrivacyAppInfo. I am not a programmer, but think that it is one or two settings of code must be changed. 

25 июля 2016 г. 13:33:51 GMT+03:00, pyler notifications@github.com пишет:

> Button in app info no, I think. But notification is fine.
> 
> ---
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pylerSM/BetterBatterySaver/issues/6#issuecomment-234918842

## 

Простите за краткость, создано в K-9 Mail.

**Comment by pylerSM** [comment]
Does it work even on 6.0?

**Comment by jumper047** [comment]
Yes, now I using it on cm13.

25 июля 2016 г. 14:32:11 GMT+03:00, pyler notifications@github.com пишет:

> Does it work even on 6.0?
> 
> ---
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pylerSM/BetterBatterySaver/issues/6#issuecomment-234929105

## 

Простите за краткость, создано в K-9 Mail.

**Comment by pylerSM** [comment]
Ok, looks fine. I will consider it now.
